rfcs/_YYYY-MM-DD-issue#-title.md

```diff
@@ -2,58 +2,56 @@
 
 One paragraph description of the change.
 
-## Motivation
-
-Context that helps readers understand the motivation behind this change.
-
-## Guide-level Proposal
+## Scope
 
-Describe your proposal in the form of a guide, as if it were being presented
-to a Vector user.
+- What will this RFC cover, and what will it not cover?
+- Link to any previous RFCs for additional context.
 
-1. Be succint and to the point!
-2. Use code examples instead of long conceptual explanations!
-3. Run your guide through grammarly or a similar tool.
+## Motivation
 
-## Doc-level Proposal
+- What is the problem?
+- What pain motivated this change?
+- Do not cover benefits of your change, that should be covered in the "Rationale" section.
 
-If this feature requires documentation or reference changes please include
-those here. Please create sections for each new page within our docs.
+## Proposal
```

Comment:
I think it is still nice to do a doc level proposal for user facing changes. Maybe we can add that as a bullet point if we are dropping the section?

Reply:
This could be an optional section for RFCs that are strongly user-facing.

Reply:
I love the enthusiasm for writing docs 😄 . It always felt burdensome to me when we were working out the initial shape of the project, but I'm happy to change it back.